### PR TITLE
New Rule (Credential phishing): Engaging language with IPFS link

### PR DESCRIPTION
**Pull request by aidenmitchell**
This is different than our existing IPFS link rule, because it uses LA to find the IPFS link. This is because some samples _redirect_ to an IPFS link to evade initial IPFS keyword detections.

### CODE REVIEW

**Comment by aidenmitchell** [comment]
Catches unflagged emails in a 30 day hunt!

File: detection-rules/suspicious_body_ipfs.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+                  )
+              // if it's in Tranco or Umbrella, still include it if it's one of these
+              or beta.linkanalysis(.).effective_url.domain.domain in $free_file_hosts
+              or beta.linkanalysis(.).effective_url.domain.root_domain in $free_subdomain_hosts
+          )
+      )
+  )
+tags: 
+  - "Natural Language Understanding"
+  - "Suspicious body"
```
Comment (jkamdjou) [inline detection-rules/suspicious_body_ipfs.yml:30]:
```suggestion
  - "Credential phishing"
```

File: detection-rules/suspicious_body_ipfs.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+name: "Suspicious body with IPFS link"
```
Comment (jkamdjou) [inline detection-rules/suspicious_body_ipfs.yml:1]:
```suggestion
name: "Credential phishing: Engaging language with IPFS link"
```

**Review by jkamdjou — APPROVED** [review]
🙌 i think this does a good job of preventing extraneous calls to NLU or LA

**Pull request merged**